Kickstarter.xcodeproj/project.pbxproj

```diff
@@ -2834,10 +2825,14 @@
 				A7ED203E1E8323E900BFFA01 /* MessageThreadsViewControllerTests.swift */,
 				D63BBCCF217E5460007E01F0 /* PaymentMethodsViewController.swift */,
 				D6F7416F21836C3D00C2DDA2 /* PaymentMethodsViewControllerTests.swift */,
-				37DEC2212257CB470051EF9B /* PledgeTableViewController.swift */,
-				37DEC2232257CB650051EF9B /* PledgeTableViewControllerTests.swift */,
-				37DEC2252257CC110051EF9B /* PledgeViewController.swift */,
-				37DEC2272257CC270051EF9B /* PledgeViewControllerTests.swift */,
+				37DEC2212257CB470051EF9B /* PledgeViewController.swift */,
```

Comment:
Could we potentially sort `Controllers` group and `View models` group in the project hierarchy inspector? Looks like they're not sorted alphabetically.